### PR TITLE
added a new subsection.

### PR DESCRIPTION
**Pull request by kqwang**
Hi, Gallego. I added a new subsection "Super-resolution imaging" for Optical Applications, as well as two new corresponding papers.

### CODE REVIEW

**Comment by kqwang** [comment]
Today, I added a new published paper for "Auto-focus"

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@
       - [Speckle Analysis](#speckle_analysis)
       - [Interferometry or Holography](#interferometry_or_holography)
       - [Wavefront sensing](#wavefront_sensing)
-
+      - [Super-resolution imaging](#super_resolution_imaging)
```
Comment (guillermogb) [inline README.md:65]: there is another subsection with a very similar name ("Super-resolution", under "Reconstruction of Visual Information"). Could we try to rename to tell them apart? This one seems to be about microscopy and super-resolution.

File: README.md [hunk 2]
```diff
@@ -2409,6 +2412,15 @@ Optics Letters, 2021.
 *[Shack-Hartmann wavefront sensing using spatial-temporal data from an event-based image sensor](https://doi.org/10.1364/OE.409682)*,  
 Optics Express, 2020.
 
+<a name="super_resolution_imaging"></a>
+### Super-resolution imaging
+- <a name="Cabriel22bioArxiv"></a>Cabriel, C., Specht, C. G. & Izeddin, I.,
```
Comment (guillermogb) [inline README.md:2417]: I think we do not use capitals in the publication part of the key.
Cabriel22bioarxiv

File: README.md [hunk 3]
```diff
@@ -2409,6 +2412,15 @@ Optics Letters, 2021.
 *[Shack-Hartmann wavefront sensing using spatial-temporal data from an event-based image sensor](https://doi.org/10.1364/OE.409682)*,  
 Optics Express, 2020.
 
+<a name="super_resolution_imaging"></a>
+### Super-resolution imaging
+- <a name="Cabriel22bioArxiv"></a>Cabriel, C., Specht, C. G. & Izeddin, I.,  
+*[Event-based vision sensor enables fast and dense single-molecule localization microscopy](https://doi.org/10.1101/2022.07.22.501162)*,  
+bioArxiv, 2022.
+- <a name="Mangalwedhekar23NN"></a>Mangalwedhekar, R., Singh, N., Thakur, C. S., Seelamantula, C. S., Jose, M., and Nair, D.,
```
Comment (guillermogb) [inline README.md:2420]: Mangalwedhekar23natnano ? (so that nn is not confused with Elsevier's Neural Networks journal)

**Review by guillermogb — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kqwang) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (kqwang) [reply README.md]: Yes, I changed the former to "image super-resolution" and the latter to "optical super-resolution". What do you think?

**Review by guillermogb — APPROVED** [review]

**Comment by guillermogb** [comment]
thanks

**Pull request merged**